packages/app-store/office365calendar/lib/CalendarService.ts

```diff
@@ -140,16 +140,18 @@ export default class Office365CalendarService implements Calendar {
         });
         const responseBody = await handleErrorsJson(response);
         return responseBody.responses.reduce(
-          (acc: BufferedBusyTime[], subResponse: { body: { value: any[] } }) =>
+          (acc: BufferedBusyTime[], subResponse: { body: { value?: any[] } }) =>
             acc.concat(
-              subResponse.body.value
-                .filter((evt) => evt.showAs !== "free" && evt.showAs !== "workingElsewhere")
-                .map((evt) => {
-                  return {
-                    start: evt.start.dateTime + "Z",
-                    end: evt.end.dateTime + "Z",
-                  };
-                })
+              subResponse.body?.value
```

Comment:
When a calendar was disconnected and getting here, subResponse.body was { error: Object } and not value thus throwing error and stopping slot available validation.